aws/resource_aws_db_snapshot_test.go

```diff
@@ -81,3 +109,64 @@ resource "aws_db_snapshot" "test" {
 	db_snapshot_identifier = "testsnapshot%d"
 }`, rInt)
 }
+
+func testAccAwsDbSnapshotConfigTags1(rInt int, tag1Key, tag1Value string) string {
+	return fmt.Sprintf(`
+resource "aws_db_instance" "bar" {
+	allocated_storage = 10
+	engine = "MySQL"
+	engine_version = "5.6.35"
+	instance_class = "db.t2.micro"
+	name = "baz"
+	password = "barbarbarbar"
+	username = "foo"
+
+	maintenance_window = "Fri:09:00-Fri:09:30"
```

Comment:
Nit: We should try to omit optional arguments that are not relevant to the acceptance test. 👍